lra/coordinator/server/src/main/java/module-info.java

```diff
@@ -27,7 +27,6 @@
     requires io.helidon.nima.http.media.jsonp;
     requires io.helidon.scheduling;
     requires io.helidon.dbclient;
-    requires io.helidon.dbclient.jdbc;
```

Comment:
Have you tried running in on module path?

Reply:
Reverted this change.